hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -286,10 +286,10 @@ msgstr ""
 "%(base_url)s%(my_virtual_machines_url)s\n"
 
 msgid "Toggle navigation"
```

Comment:
What is "Toggle navigation" for? Trying to understand the relation with "Konfiguration/Umschalten."

Reply:
That seems to come from the navbar, probably @tiwariav can have an explanation for that